README.md

```diff
@@ -0,0 +1,95 @@
+# UrContacts
+> A contact list application
+
+UrContacts is a React contact list app built with full CRUD capabilities. It uses json-server to locally store your data in a JSON file at the backend. Because your data persists on a locally hosted database, your data is completely private.
+
+## Installation and Build
+
+1. Clone GitHub repo into a desired local root directory: 
+    - HTTPS: https://github.com/dyazdani/contact-list.git
+    - SSH: `git@github.com:dyazdani/contact-list.git`
+    - GitHub CLI: `gh repo clone dyazdani/contact-list` 
+
+2. `cd contact-list`
+
+3. `npm install` 
+
+4. `npm run build`
+
+5. `npm start`
+
+
+## Features
+
+UrContacts provides functionality for creating, updating, deleting, and searching for contacts in a list stored as a JSON file on a locally hosted json-server. The app uses async calls to the json-server using the fetch API, and data that you add, update, or delete persist in the json-server in the `db.json` file. The app uses the React Router package to create routes to different application features while maintaining a single-page application experience.
+
+### Technologies Used
+- React
+- TypeScript
+- Vite
+- React Router
+    UrContacts uses React Router to route to various URLs in on your browser: 
+
+    /                               => Home page
+    /contacts                       => contact list
+    /contacts/{contactID}           => contact by id
+    /create                         => create new contact
+    /contacts/{contactID}/update    => update contact
+- [json-server][json-server] 
+
+
+## User Experience
+### Home Page `/`
+- From the home page, you will be able to see how many contacts are currently in your list.
+- If there are no contacts, you will be prompted to create a new contact. 
+- Click on "Go to UrContacts List" to view your contact list (`/contacts`). 
```

Comment:
A common technical documentation style included in many techdocs style guides (I think it may come from Cisco originally but I'm not sure, I've seen it in so many style guides at this point) is bold for anything that is an actual term from the UI. I think it helps improve readability

```suggestion
- Click **Go to UrContacts List** to view your contact list (`/contacts`). 
```